.npmrc

```diff
@@ -0,0 +1,2 @@
+shamefully-hoist=true
```

Comment:
I believe that this came with the official nuxt module project template. IIRC it's a nuxt + pnpm problem, going to confirm and change if appropriate.

Reply:
Update: correct, it was a part of the nuxt starter, since then removed (see https://github.com/nuxt/cli/issues/316). Going to remove .npmrc completely.